source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -166,7 +178,7 @@ def create_blob_put_request(uri, msg, request_id, file_path = nil)
       req.body = msg
       return req
     rescue OMS::RetryRequestException => e
-        OMS::Log.error_once("HTTP error for Request-ID: #{request_id} Error: #{e}")
+        Log.error_once("HTTP error for Request-ID: #{request_id} Error: #{e}")
```

Comment:
Please keep the `OMS::` scope on this call. FluentD's log does not have an error_once method, and I've seen an error thrown because of a call missing this scope.